src/style.css

```
@@ -1 +1,37 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  gap: 4px;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-size: 13px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--stars-1 .stars__star:nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--stars-2 .stars__star:nth-child(-n + 2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--stars-3 .stars__star:nth-child(-n + 3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--stars-4 .stars__star:nth-child(-n + 4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--stars-5 .stars__star:nth-child(-n + 5) {
+  background-image: url(images/star-active.svg);
+}
```

```suggestion
.stars--stars-1 .stars__star:nth-child(1),
.stars--stars-2 .stars__star:nth-child(-n + 2),
.stars--stars-3 .stars__star:nth-child(-n + 3),
.stars--stars-4 .stars__star:nth-child(-n + 4),
.stars--stars-5 .stars__star:nth-child(-n + 5) {
  background-image: url(images/star-active.svg);
}
```

Combine selectors using comma